pkg/pkg.mk

```diff
@@ -13,6 +13,7 @@ GIT_APPLY_PATCHES:=if test -d "$(PKG_DIR)"/patches; then \
 	fi
 
 $(PKG_BUILDDIR)/.git-downloaded:
+	rm -Rf $(PKG_BUILDDIR)
```

Comment:
"mkdir -p" creates not only the target directory, but also the tree leading to it. so here it's needed even if the checkout would also create the target directory.

-----Original Message-----
From: "Cenk Gündoğan" <notifications@github.com>
To: RIOT-OS/RIOT <RIOT@noreply.github.com>
Cc: Kaspar Schleiser <kaspar@schleiser.de>
Sent: Fr., 01 Apr. 2016 13:59
Subject: Re: [RIOT-OS/RIOT] make: pkg: remove pkg source dir before checkout (#5219)

> @@ -13,6 +13,7 @@ GIT_APPLY_PATCHES:=if test -d "$(PKG_DIR)"/patches; then \
>  	fi
>  
>  $(PKG_BUILDDIR)/.git-downloaded:
> +	rm -Rf $(PKG_BUILDDIR)

>Would have bet, that the -p in the mkdir below has the same effect

On my system, `mkdir -p` has luckily not the same effect as `rm-Rf`

---
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/RIOT-OS/RIOT/pull/5219/files/29e7ea454a948ea5cf51f3c5b0a1d6a05bc5df8b#r58195810